tools/prom-rule-ci/verify-rules.go

```diff
@@ -0,0 +1,101 @@
+package main
```

Comment:
Is this file should be added for all operators?

Reply:
here they already had the rules unit tests, on kubevirt they already had something similar, so it really depends